### PR TITLE
Add an entry point so this system can work without locally installing…

### LINKED ISSUE #110: Python package multi config not separate
Ask ben for details

### PR DESCRIPTION
**Pull request by Sean1572**
… the package

### CODE REVIEW

**Review by benjamin-cates — COMMENTED** [review]
What weird directory issues did you have? Cause now this will not work unless you run it at the project root

**Comment by benjamin-cates** [comment]
Okay status of the entry_point branch:

There are three options to running the code:
1. Run `entry.py` from the root directory (works)
2. Install with `pip install -e .` and be able to run anywhere with `python -m pyha_analyzer.train`
3. Install with `pip install .` and be able to run anywhere with `python -m pyha_analyzer.train` (however not recommended because file edits will not be changed)

**Comment by benjamin-cates** [comment]
Config.yml and default_config.yml are now in the package directory

**Review by marco-bazzani — DISMISSED** [review]
Groovy

File: pyha_analyzer/default_config.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ manual_id_col: "MANUAL ID"
 # Device Settings
 device: auto # Options: cuda, cpu, auto 
 prepros_device: cpu # Options: cuda, CPU
-
+agent_run_count: 5
```
Comment (sprestrelski) [inline pyha_analyzer/default_config.yml]: what is this parameter

**Review by sprestrelski — COMMENTED** [review]
1. Using `entry.py`, setting logging to false in config.yml doesn't seem to work. However, I'm able to run training using it.
2. Using `pip install -e .` works with both `python -m pyha_analyzer.train` from the root directory and `python train.py` from the pyha_analyzer directory, but the first method gives me the following RuntimeWarning. Not sure if there's something wrong with my setup.
```
/home/sprestrelski/miniconda3/envs/asid/lib/python3.8/runpy.py:127: RuntimeWarning: 'pyha_analyzer.train' found in sys.modules after import of package 'pyha_analyzer', but prior to execution of 'pyha_analyzer.train'; this may result in unpredictable behaviour
  warn(RuntimeWarning(msg))
```

**Comment by sprestrelski** [comment]
pyright dislikes `importlib_resources` :/

File: pyha_analyzer/default_config.yml (hunk 1 above)
Comment (sprestrelski) [reply pyha_analyzer/default_config.yml]: nvm, documented in another branch

**Comment by benjamin-cates** [comment]
Okay I fixed the logging always being on issue, it has in_sweep=True as a default in train.main.

I also found another bug, if you build it with `pip install .` and then generate a config file, it won't delete the `pyha_analyzer` folder in the conda packages directory. Importlib still thinks that the directory is there even though it is empty and not being used, so it can't find `default_config.yml`. Moral of the story is don't install it as a build and then uninstall it without deleting that directory

**Review by sprestrelski — APPROVED** [review]
No logging works :)

**Pull request merged**